### PR TITLE
Fix some minor typos in readme

### PR DESCRIPTION
**Pull request by chraibi**
Some typos  in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,16 +67,16 @@ To run all tests at once call `ctest -j$(nproc)` in the build folder.
 
 ### Running jps
 
-To run *jps* jpscore.so location needs to be on the `PYTHONPATH`, e.g.:
+To run *jps* jpscore's location needs to be in the `PYTHONPATH`, e.g.:
```
Comment (schroedtert) [inline README.md:70]: `jpscore.so` is not a type but the name of the library on unix systems

```suggestion
To run *jps* the location of the jpscore library (e.g., on Linux `jpscore.so`) needs to be in the `PYTHONPATH`, e.g.:
```

**Review by schroedtert — CHANGES_REQUESTED** [review]

**Review by schroedtert — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -67,16 +67,16 @@ To run all tests at once call `ctest -j$(nproc)` in the build folder.
 
 ### Running jps
 
-To run *jps* jpscore.so location needs to be on the `PYTHONPATH`, e.g.:
+To run *jps* the location of the jpscore library (e.g., on Linux `jpscore.so`) needs to be in the `PYTHONPATH`, e.g.:
 
 ```bash
 PYTHONPATH=~/simulation-build/bin ./jps/jps --help
```
Comment (JetteSchumann) [inline README.md:73]:
```suggestion
PYTHONPATH=~/simulator-build/bin ./jps/jps --help
```

File: README.md [hunk 3]
```diff
@@ -67,16 +67,16 @@ To run all tests at once call `ctest -j$(nproc)` in the build folder.
...
 ### Repository Layout
 
 The repository layout follows the high level architecture of the *jps* tool. At
-the core is a C++ library to do the pedestrian simulation. This core is wrapped
-with pybind11 to allow use of the simulation library from python. Lastly there
+the core is a C++ library for pedestrian simulation. This core is wrapped
+with `pybind11` to allow use of the simulation library from python. Lastly there
 is the actual *jps* application written in python. The python layer is used to
 handle all the non-resource / compute intensive parts of the application, i.e.
 initial setup, file handling etc.
```
Comment (JetteSchumann) [inline README.md:82]: Could you please correct line 108 as well?

**Review by JetteSchumann — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (chraibi) [reply README.md:82]: Done!

**Review by zeroset — APPROVED** [review]
LGTM

**Review by JetteSchumann — APPROVED** [review]

**Pull request merged**